Source/Mosa.Platform.x64/Intrinsic/CpuIdEax.cs

```diff
@@ -15,9 +15,9 @@ private static void CpuIdEax(Context context, MethodCompiler methodCompiler)
 		{
 			var result = context.Result;
 			var operand = context.Operand1;
-			var eax = Operand.CreateCPURegister(methodCompiler.TypeSystem.BuiltIn.I4, CPURegister.EAX);
-			var ecx = Operand.CreateCPURegister(methodCompiler.TypeSystem.BuiltIn.I4, CPURegister.ECX);
-			var reg = Operand.CreateCPURegister(methodCompiler.TypeSystem.BuiltIn.I4, CPURegister.EAX);
+			var eax = Operand.CreateCPURegister(methodCompiler.TypeSystem.BuiltIn.I4, CPURegister.R1);
+			var ecx = Operand.CreateCPURegister(methodCompiler.TypeSystem.BuiltIn.I4, CPURegister.R1);
+			var reg = Operand.CreateCPURegister(methodCompiler.TypeSystem.BuiltIn.I4, CPURegister.R1);
```

Comment:
Maybe this should use `RAX` specifically for both register name.
Same for the rest of the intrinsic files